documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/http/server-http-handler.adoc

```diff
@@ -16,21 +16,14 @@
 
 An `org.eclipse.jetty.server.Handler` is the component that processes incoming HTTP requests and eventually produces HTTP responses.
 
-``Handler``s can be organized in different ways:
+``Handler``s can process the HTTP request themselves, or they can be a ``Handler.Container``s that delegate HTTP request processing to one or more contained ``Handler``s.  This allows ``Handler``s to be organised as a tree comprised of:
```

Comment:
All sentences on a new line, so `This allows...` needs to be on a new line.

Reply:
done.